ts/menu_ts.ts

```diff
@@ -1068,7 +1074,7 @@ class ShowContent extends Content {
 
       case "filter":
         showPreview(true)
-        sortTable(0)
+        sortTable(1)
```

Comment:
I wanted to have the table sorted by channel number, but with how sortTable works, we'd lose some rows if any filters had duplicate Starting Number....and I didn't want to overhaul sort at this point, so I left it sorting by Filter Name.